app/views/devise/registrations/new.html.haml

```diff
@@ -1,38 +1,40 @@
-%br
-%h4 
-  Thank you for your interest! Becoming a member provides you access to our newsletters.
-  In order to also view our premium content, please become a member first (below), then subscribe.
-  
-%br
-%h2 Register
-
-= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-
-  = devise_error_messages!
-
-  .field
-    = f.label :name
-    %br/
-    = f.text_field :name, autofocus: true, autocomplete: 'name'
-
-  .field
-    = f.label :email 
-    %br/
-    = f.email_field :email, autofocus: true, autocomplete: 'email'
-
-  .field
-    = f.label :password
-    - if @minimum_password_length
-      %em
-        (#{@minimum_password_length} characters minimum)
-    %br/
-    = f.password_field :password, autocomplete: 'new-password'
-
-  .field
-    = f.label :password_confirmation
-    %br/
-    = f.password_field :password_confirmation, autocomplete: 'new-password'
-  .actions
-    = f.submit 'Sign up'
-
-= render 'devise/shared/links'
\ No newline at end of file
+
+.signup_container
+  .signup_header.text-red-darkest.text-center.mb-16.text-lg
+    Thank you for your interest! Becoming a member provides you access to our newsletters 
+    and allows you to comment on the articles. In order to also view our premium content, 
+    please become a member first (below), then proceed to become a subscriber.
+
```

Comment:
Not coding related but maybe mention that becoming a subscriber will also give you access to mobile content.

Reply:
Same as above